test/specs/openapi/test_examples.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+import src.schemathesis as schemathesis
```

Comment:
Hm, I believe, that `import schemathesis` should work if it is tested via `tox` / a shell that has `PYTHONPATH` set to the `src` dir. It is done this way in other tests